internal/data/data.go

```diff
@@ -0,0 +1,47 @@
+package data
+
+import "encoding/base64"
+
+const TestPngImageB64 = `
```

Comment:
`go embed` isn't available until 1.16, and the go.mod is at 1.13, I didn't want to bump (unless it is desirable to embed this blob from a seperate file)

Reply:
`go embed` would be preferable here, as it's much easier to check that the PNG is valid as you can just open it rather than just trust that this base-64 is a PNG. I'm happy for this to be bumped to 1.16